### PR TITLE
feat(supported-version): add support for v2.4.5

### PR DESCRIPTION
**Pull request by damienwebdev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/github-actions-magento2/blob/main/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
We don't have v2.4.5 in the supported version matrix.

Requires https://github.com/mage-os/generate-mirror-repo-js/issues/4

## What is the new behavior?
We now do.

## Does this PR introduce a breaking change?
- [x] Yes
- [ ] No

v2.4.5 has breaking changes, and is now in `supported-version` `currently-supported` and is `latest`. Your code may have a breaking change.

### CODE REVIEW

**Comment by damienwebdev** [comment]
@vinai We do we know when the mirror will have the latest tags?

**Comment by Vinai** [comment]
The upstream repos seem to have the new tags now, so I've just triggered the action to synchronize the mirror repos and then I'll run the mirror build script locally to see if it works ootb or if it needs some tweaks.

**Review by ihor-sviziev — DISMISSED** [review]
Please disable this line `composer require monolog/monolog:"!=2.7.0" --no-update` for:
- magento/product-community-edition 2.4.4-p1 and above
- magento/product-community-edition 2.4.5 and above

File: supported-version/src/versions/individual.json [hunk 1]
```diff
@@ -11,6 +11,18 @@
         "nginx": "nginx:1.18",
         "os": "ubuntu-18.04"
     },
+    "magento/project-community-edition:2.3.7-p4": {
+        "magento": "magento/project-community-edition:2.3.7-p3",
```
Comment (ihor-sviziev) [inline supported-version/src/versions/individual.json:15]:
```suggestion
        "magento": "magento/project-community-edition:2.3.7-p4",
```

File: supported-version/src/versions/individual.json [hunk 2]
```diff
@@ -131,6 +143,18 @@
         "nginx": "nginx:1.18",
         "os": "ubuntu-latest"
     },
+    "magento/project-community-edition:2.4.3-p3": {
+        "magento": "magento/project-community-edition:2.4.3-p2",
```
Comment (ihor-sviziev) [inline supported-version/src/versions/individual.json:147]:
```suggestion
        "magento": "magento/project-community-edition:2.4.3-p3",
```

File: supported-version/src/versions/individual.json (hunk 1 above)
Comment (damienwebdev) [reply supported-version/src/versions/individual.json]: Fixed, ty for the find!

File: supported-version/src/versions/individual.json (hunk 2 above)
Comment (damienwebdev) [reply supported-version/src/versions/individual.json]: Fixed, ty for the find!

**Review by ihor-sviziev — APPROVED** [review]

**Pull request merged**